custom/icds_reports/utils/usage_data_pull.py

```diff
@@ -0,0 +1,88 @@
+from custom.icds_reports.models.util import ICDSAuditEntryRecord
+from custom.icds_reports.models.aggregate import  AggAwc
+import re
+from datetime import date
+from corehq.apps.users.models import CommCareUser
+import csv
+
+
+
+
+user_objects = {}
+dashboard_uname_rx = re.compile(r'^\d*\.[a-zA-Z]*@.*')
+
+
+date_setp = [(date(2019,9,2),date(2019,9,8)),(date(2019,9,9),date(2019,9,15)),(date(2019,9,16),date(2019,9,22)),(date(2019,9,23),date(2019,9,29)),(date(2019,9,30),date(2019,9,30))]
```

Comment:
These are really hard to read. It took me quite a while to realize its a list of tuples instead of just a list of dates